recipes/list_items/contact_info/contact_info.vue

```diff
@@ -133,6 +172,19 @@ export default {
       type: String,
       default: null,
     },
+
+    /**
+     * Showing multiple avatars in contact info.
+     * The props of array items are:
+     * `src` - avatar image url
+     * `initials` - Initial letters to display in avatar if `src` is empty.
+     * `text` - cover text that showing over avatar
+     * `icon` - cover icon that showing over avatar
```

Comment:
"text that overlays the avatar"

"icon that overlays the avatar"

Reply:
I think if we want to support showing some info for each user (for example, admin listen in group calls) in the future, we don't need change the `DtRecipeContactInfo`.
<img width="390" alt="multiple-avatars" src="https://user-images.githubusercontent.com/86271164/230105843-3fe3eb39-f464-436f-a408-1b7c2b60a1e5.png">

Reply:
okay yeah if that's a valid case, definitely leave it as is.